traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -36,7 +39,7 @@ import (
 
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/deliveryservice"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/tenant"
-	"github.com/go-ozzo/ozzo-validation"
+	validation "github.com/go-ozzo/ozzo-validation"
```

Comment:
why?

Reply:
just matched how its being called in code but 🤷‍♂ ill change it back

Reply:
I've seen that in few places, but the package is named `validation` already, so idk why. Maybe it used to be necessary when the package name didn't match the terminating directory name?

Reply:
+1 on removing the alias. Yeah, we have a bunch of places that do it, I'm guessing whoever added the library first didn't realize Go follows the package name, not the directory; and everyone else just copied it. If it's an idiom, IMO it's an antipattern and we should remove them.